infinitest-lib/.gitignore

```diff
@@ -0,0 +1 @@
+/.factorypath
```

Comment:
This is an eclipse configuration file. It configures the compiler for annotation processing.
It is ignored because it is generated when importing maven files into eclipse.